lib_xtcp/module_build_info

```diff
@@ -3,18 +3,16 @@ VERSION = 6.2.0
 DEPENDENT_MODULES = lib_ethernet(>=4.1.0) lib_otpinfo(>=2.2.1) lib_random(>=1.3.0) lib_xassert(>=4.3.2) lib_logging(>=3.4.0)
 
 # Build flags
-MODULE_XCC_FLAGS = $(XCC_FLAGS) -g -O3
+MODULE_XCC_FLAGS = $(XCC_FLAGS) -g -Os  -mno-dual-issue
```

Comment:
-Os is non-dual issue anyway. You can get all of the goodness from -O3 and keep the -mno-dual-issue flag.